pages/home/home.css

```diff
@@ -0,0 +1,183 @@
+.explore-categories {
+  background-color: var(--category-background-color);
+  padding-top: 4rem;
+  padding-bottom: 6rem;
+  text-align: center;
+}
+.title-block {
+  padding: 2rem 0 1rem 0;
+  font-size: 2.8rem;
+  line-height: 3.2rem;
+  letter-spacing: 0.5px;
+  color: var(--primary-font-color);
+  text-align: center;
+  text-transform: capitalize;
+  position: relative;
+  margin-bottom: 3rem;
+}
+.title-block:after {
+  position: absolute;
+  content: "";
+  width: 9rem;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: 0 auto;
+  border-bottom: 1px solid var(--primary-color);
+}
+.explore-categories-row {
+  display: flex;
+  max-width: 85%;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  margin: 0 auto;
+  flex-wrap: wrap;
+}
+.category {
+  flex: 1;
+  margin: 1.5rem;
+  cursor: pointer;
+  text-decoration: none;
+}
+.category-image {
+  height: 6rem;
+  background-repeat: no-repeat;
+  display: inline-block;
+  background-position: bottom;
+  background-size: contain;
+  max-width: 6rem;
+  width: 100%;
+  margin-bottom: 5px;
+}
+.category-name {
+  color: var(--primary-font-color);
+  font-size: 1.5rem;
+  line-height: 2rem;
+  padding: 0;
+}
+
+.featured-product-container {
+  padding-left: 5rem;
+  padding-right: 5rem;
```

Comment:
Change- You can use shorthand property like
padding: 0 5rem;

Reply:
Done